src/templates/docs-template.js

```diff
@@ -44,6 +44,18 @@ const DocsTemplate = ({ data }) => {
                 Keycloak
               </NavItem>
             </Nav>
+            <h5>Hawtio in Containers</h5>
```

Comment:
We should simply write `Kubernetes/OpenShift` here. "Containers" is not the best word for hawtio-online.